### PR TITLE
Adding permission manager tool

### PR DESCRIPTION
**Pull request by milindchawre**
Adding permission manager tool to civo marketplace.

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [x] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [x] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?

**ScreenShots**
image

### CODE REVIEW

File: permission-manager/install.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+kind: Secret
+metadata:
+  name: permission-manager
+  namespace: permission-manager
+type: Opaque
+stringData:
+  PORT: "4000" # port where server is exposed
+  CLUSTER_NAME: "${CLUSTER_NAME}" # name of the cluster to use in the generated kubeconfig file
+  CONTROL_PLANE_ADDRESS: "https://${MASTER_IP}:6443" # full address of the control plane to use in the generated kubeconfig file
+  BASIC_AUTH_PASSWORD: $(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 10 ; echo '') # password used by basic auth (username is "admin")
```
Comment (alejandrojnm) [inline permission-manager/install.sh:14]: why not use CIVO: ALPHANUMERIC, this way it will be reflected in the user interface

Comment (saiyam1814) [reply permission-manager/install.sh:14]: @milindchawre can you fix this ? So that we can merge this ?

**Comment by saiyam1814** [comment]
BTW @milindchawre did you saw https://blog.kubernauts.io/permission-manager-rbac-management-for-kubernetes-ed46c2f38cfb ??

**Comment by saiyam1814** [comment]
lgtm

**Review by milindchawre — COMMENTED** [review]
> 
> 
> BTW @milindchawre did you saw https://blog.kubernauts.io/permission-manager-rbac-management-for-kubernetes-ed46c2f38cfb ??

Yeah. That was the first blog I referred few months back. I also added it to this repo https://github.com/mhausenblas/rbac.dev/pull/9 😉

**Comment by milindchawre** [comment]
> 
> BTW @milindchawre did you saw https://blog.kubernauts.io/permission-manager-rbac-management-for-kubernetes-ed46c2f38cfb ??

Yes @saiyam1814  That was the first blog I referred few months back. I also added it to this repo https://github.com/mhausenblas/rbac.dev/pull/9 😉

**Pull request merged**